pkg/awsv1alpha1/accounts.go

```diff
@@ -0,0 +1,62 @@
+package awsv1alpha1
+
+import (
+	"context"
+	"fmt"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	awsCredsSecretIDKey     = "aws_access_key_id"
+	awsCredsSecretAccessKey = "aws_secret_access_key"
+	namespace               = "aws-account-shredder"             // change the namespace according to your environment. this is the namespace, from where secret has to retreived from
+	secretName              = "aws-account-shredder-credentials" // the name of the secret to be read
+)
+
+func GetAcoountCRsToReset(ctx context.Context, cli client.Client) ([]string, error) {
```

Comment:
Small type in GetAcoount

Reply:
Also, my bad, this should be `GetAccountIDsToReset` since you're returning Account IDs, and not Account CR names.